test/resources/case32_secret_stringdata/case32_create_secret.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: case32config
+spec:
+  evaluationInterval:
+    compliant: 15s
+    noncompliant: 15s
```

Comment:
Nit: you could just leave the evalauationInterval unset, the default is effectively 10 seconds anyway.